cmd/bfgd/bfgd.go

```diff
@@ -31,60 +31,34 @@ var (
 
 	cfg = bfg.NewDefaultConfig()
 	cm  = config.CfgMap{
-		"BFG_EXBTC_ADDRESS": config.Config{
-			Value:        &cfg.EXBTCAddress,
-			DefaultValue: "localhost:18001",
-			Help:         "electrs endpoint",
+		"BFG_BITCOIN_SOURCE": config.Config{
+			Value:        &cfg.BitcoinSource,
+			DefaultValue: "blockstream",
+			Help:         "which bitcoin source of truth is used (blockstream and tbc)",
```

Comment:
```suggestion
			Help:         "which bitcoin source of truth is used (options: blockstream, tbc)",
```

Reply:
we don't do this in the other daemons here.